File: DIC/WebServices/openfaas/python-code/hello/handler.py
#from helper import helper
import time
import json

def handle(args):
    startTime = time.time()

    if 'name' in args:
       name = args['name']
    else:
       name = "stranger"
    greeting = "Hello from helper.py, " + name + "!"

    return json.dumps({'token': greeting, 'startTime': startTime})

File: DIC/WebServices/openfaas/go-code/hello/handler.go
package function

import (
//	"fmt"
	"time"
	"encoding/json"
)

// Main function for the action
func Handle(req []byte) string {
	now:= time.Now()
	startTime := now.UnixNano() / 1e6

	name := "stranger"
	
	msg := make(map[string]interface{})
	msg["msg"] = "Hello, " + name + "!"
	msg["startTime"] =startTime

	mjson,_ :=json.Marshal(msg)
	mString :=string(mjson)
	
	return mString
}